### DISCUSSION TITLE
Discord link in the footer of the site is broken

### DISCUSSION THREAD

**Discussion by kkomelin** [heroui-inc/heroui#193] [Feedback]
The link:
https://discord.gg/9b6yyZKmH4

The error message screenshot:
https://imgur.com/xBpg3Oy

**Comment by jrgarciadev** [comment]
Hey @kkomelin thanks for reporting it, the link is working for me 🤔 , here's another invitation link -> https://discord.gg/PXPverC6wh

**Comment by kkomelin** [comment]
Thanks @jrgarciadev. There is probably an issue with my discord account, so I contacted their support.
